securedrop/server.py

```diff
@@ -185,18 +185,17 @@ def add_contact(self, email, contact_name, contact_email):
         self.write_json()
         return ""
 
-    def list_contacts(self, email):
+    def get_contacts(self, email):
         if not email:
             return "Invalid email address"
         email_hash = SHA256.new(email.encode()).hexdigest()
-        users_keys_list = self.users.keys()
-        user = dict()
+        user_contacts_dict = dict()
```

Comment:
You can just replace everything past here with:
```python
return self.users[email_hash].contacts if email_hash in self.users else dict()
```

Reply:
Changed.